contracts/mocks/EnumerableMapMock.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity ^0.6.0;
+
+import "../utils/EnumerableMap.sol";
+
+contract EnumerableMapMock {
+    using EnumerableMap for EnumerableMap.UintToAddressMap;
+
+    event TransactionResult(bool result);
```

Comment:
Really bad name. :stuck_out_tongue: What about `WasPresent`...?

Reply:
I changed the name to `OperationResult`. The return value is closer to `success`: it returns whether the call actually did anything or not, which for `add` happens when the key was _not_ present.